third_party/blink/renderer/platform/fonts/shaping/harfbuzz_shaper.cc

```diff
@@ -789,7 +789,16 @@ void HarfBuzzShaper::ShapeSegment(
 
       current_font_data_for_range_set =
           fallback_iterator.Next(fallback_chars_hint);
-      if (!current_font_data_for_range_set->FontData()) {
+
+      // [RecordReplay] See RUN-1302
```

Comment:
We probably might want to come up with a unified format to refer to bug fix explanations in Linear. I personally would remove "See RUN-1302" since its rendundant, and then just move the link to the bottom of the comment.

Reply:
Fixed up the comments a bit to make them less redundant.